src/pocketmine/level/Level.php

```diff
@@ -1756,7 +1756,11 @@ public function useItemOn(Vector3 $vector, Item &$item, int $face, float $fx = 0
 
 			Tile::createTile("Sign", $this->getChunk($block->x >> 4, $block->z >> 4), $nbt);
 		}
-		$item->setCount($item->getCount() - 1);
+		if ($player != null && $player->isCreative()) {
+			$item->setCount($item->getCount());
```

Comment:
https://github.com/PocketMine/PocketMine-MP/blob/master/src/pocketmine/item/Item.php#L1038

Now, do you expect simply running `$this->count = $this->count;` has a point at all?